resources/istio/files/uninstall.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash -e
+if [ -f "/etc/istio/overrides.yaml" ]; then
+  #New way: just merge default IstioControlPlane definition with a user-provided one.
+  yq merge -a -x /etc/istio/config.yaml /etc/istio/overrides.yaml > /etc/combo.yaml
+
+  istioctl manifest generate -f /etc/combo.yaml | kubectl delete -f -
+  kubectl create cm "${CONFIGMAP_NAME}" -n "${NAMESPACE}"
+else
+  #Old way: apply single-value Helm overrides using `istioctl --set "key=val"`
+  overrides=$(kubectl get cm --all-namespaces -l "installer=overrides,component=istio" -o go-template --template='{{ range .items }}{{ range $key, $value := .data }}{{ if ne $key "kyma_istio_control_plane" }}{{ printf "%s: %s\n" $key . }}{{ end }}{{ end }}{{ end }}' )
+  overrides_transformed=""
+
```

Comment:
Note : I did not extract this to common utils, support for 'old' helm overrides will be deleted once we will upgrade istio.